cmd/edit/cluster/clusterregistryconfig_test.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Edit cluster", func() {
+	Context("BuildAdditionalTrustedCAFromInputFile", func() {
+		It("OK: shoud work with proper json format", func() {
+			caPath := "specRegistryAdditionalTrustedCa.json"
+			ca, err := BuildAdditionalTrustedCAFromInputFile(caPath)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(ca["userRegistry.io"]).To(
+				Equal("-----BEGIN CERTIFICATE-----\n/abc\n-----END CERTIFICATE-----"))
+
+		})
+		It("KO: fail if the spec file is invalid", func() {
+			_, err := BuildAdditionalTrustedCAFromInputFile("not-exist")
+			Expect(err).To(HaveOccurred())
+			Expect(err).To(MatchError(
+				"Expected a valid additional trusted certificate spec file: open not-exist: no such file or directory"))
```

Comment:
When using `MatchError`, `Expect(err).To(HaveOccurred())` is not needed: `MatchError` will fail if the `err` is nil
```suggestion
			Expect(err).To(MatchError(
				"Expected a valid additional trusted certificate spec file: open not-exist: no such file or directory"))
```

Reply:
Thanks @ziccardi 👍